osu.Game/Skinning/Editor/SkinComponentToolbox.cs

```diff
@@ -92,6 +92,9 @@ private static ToolboxComponentButton attemptAddComponent(Type type)
 
         private class ToolboxComponentButton : OsuButton
         {
+            protected override bool ReceivePositionalInputAtSubTree(Vector2 screenSpacePos) => false;
+            public override bool PropagateNonPositionalInputSubTree => false;
```

Comment:
I assume this is just to stop children from receiving input? Does:
`protected override bool ShouldBeConsideredForInput(Drawable child)` work?

Reply:
It does work. Is that preferred?

Reply:
It's used in other cases, so I think so.

For me, I find this hard to read as is - the `ReceivePositionalInputAtSubTree` line makes it seem like it should stop `OnClick()` input so I don't know why it's working at all.

Reply:
As mentioned on the framework side, there's a disconnect between `ReceivePositionalInputAtSubTree` and `PropagatePositionalInputSubTree` which probably needs to be resolved..

Reply:
Ah, I was glossing over the `SubTree` suffix... Still prefer using `ShouldBeConsideredForInput`...